components/shared/SkeletonLoader.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Nice call to pull this into it's own component. Great start to help us blow this out into skeleton loaders we can use on all the things.